erlang/atomvm/Portfile

```
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+PortGroup       cmake 1.1
+
+github.setup    atomvm atomvm v0.6.0-alpha.0
```

`github.setup`'s format is `<author> <project> <version> <version_prefix> <version_suffix>`.

So in this case, the correct representation would be:

```suggestion
github.setup    atomvm atomvm 0.6.0-alpha.0 v
```